types/less/index.d.ts

```diff
@@ -293,12 +216,93 @@ interface LessStatic {
 
     watch(): void;
 
-    FileManager: typeof Less.FileManager;
-    PluginManager: typeof Less.PluginManager;
+    FileManager: typeof FileManager;
+    PluginManager: typeof PluginManager;
 }
 
-declare module "less" {
-    export = less;
+declare class PluginManager {
```

Comment:
Why did these classes move outside the namespace? It doesn't seem like it'd be required at all for the PR, and it's making the diff pretty hard to understand.